packages/components/activity/sections/manage-content.tsx

```diff
@@ -39,8 +39,11 @@ export const ManageContent: FC<ManageContentProps> = ({
   const { acter } = useActer()
   const { user } = useUser()
 
+  const tabs = [MEMBERS, INVITE]
   const isAdmin = userHasRoleOnActer(user, ActerConnectionRole.ADMIN, acter)
-  const tabs = [MEMBERS, INVITE, isAdmin && SETTINGS]
```

Comment:
@judemusyoki  if you want to add element conditionally this is not right way because if that condition become `false` it adds `undefined` as element to the array. so you need to add this way
`if (isAdmin) {
  tabs.push(SETTINGS)
  }`

Reply:
Gotcha, thanks for the advice